otbenchmark/_SensitivityConvergence.py

```
@@ -243,50 +327,39 @@ def plotConvergenceGrid(
-        sample_size_final = np.max(sample_size_table)
-        sample_size_log_array = np.logspace(
-            np.log10(sample_size_initial), np.log10(sample_size_final)
-        )
-        sampleSizeArray = [int(n) for n in sample_size_log_array]
-        expectedConvergence = [1.0 / np.sqrt(n) for n in sampleSizeArray]
+            sampleSizeTable,
+            firstOrderTable,
+            totalOrderTable,
+            sampleSizeArray,
+            expectedConvergence,
+            referenceLegend,
```

Ruff (0.15.15)

[warning] Unpacked variable `referenceLegend` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)